example.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.Linq;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.ServerWide;
+using Raven.Client.ServerWide.Operations;
+
+namespace HelloWorld
+{
+    internal static class Program
+    {
+        private const string DatabaseName = "MyFirstDB";
+
+        private class User
+        {
+            public string Name;
+            public string Domain;
+            public DateTime LastLogin;
+        }
+        
+        private class DomainFullTextSearch : AbstractIndexCreationTask<User>
+        {
+            public DomainFullTextSearch()
+            {
+                Map = users => from user in users
+                    select new
+                    {
+                        Name = user.Name,
+                        Domain = user.Domain,
+                        LastLogin = user.LastLogin
+                    };
+                
+                Indexes.Add(x => x.Domain, FieldIndexing.Search);
+                Indexes.Add(x => x.LastLogin, FieldIndexing.Search);
+            }
+            
+            
+        }
+
+        private static void Main()
+        {
+            using (var documentStore = new DocumentStore
+            {
+                Urls = new[] {"http://localhost:50000"},
```

Comment:
This is going to almost certainly conflict with the default Kestrel one, no?
Why no `8080`?